src/explore-education-statistics-common/src/components/ContentHtml.tsx

```diff
@@ -87,11 +88,11 @@ export default function ContentHtml({
         return !node.attribs.href.includes(
           'explore-education-statistics.service.gov.uk',
         ) && typeof node.attribs['data-featured-table'] === 'undefined' ? (
-          <a href={url} target="_blank" rel="noopener noreferrer">
+          <AnchorHtml href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
Do we need redirects for external links? We don't have redirects for external URLs, so this doesn't seem necessary